packages/fluentui/react-icons-northstar/src/components/Survey20pxIcon.tsx

```diff
@@ -7,14 +7,14 @@ import { iconClassNames } from '../utils/iconClassNames';
 const Survey20pxIcon = createSvgIcon({
```

Comment:
Please remove this icon, as it is not exported. If it should be consider changing the name again not to have `px` in the name